src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests.csproj

```diff
@@ -15,6 +15,7 @@
   <ItemGroup>
     <PackageReference Include="Hl7.Fhir.R4" Version="2.0.3" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" NoWarn="NU1605" />
```

Comment:
Disable warning intended?

Reply:
Yes. It was intentionally disabled. I kept the current version because the latest version of Newtonsoft has some breaking change.